stdlib/public/SwiftShims/swift/shims/_SwiftConcurrency.h

```diff
@@ -30,11 +32,6 @@ typedef struct _SwiftContext {
 } // namespace swift
 #endif
 
-#ifdef __cplusplus
-extern "C" [[noreturn]]
-#endif
-void exit(int);
-
 #define EXIT_SUCCESS 0
```

Comment:
I think `EXIT_SUCCESS` can also be removed since we now import `stdlib.h` where it is defined.